### ISSUE TITLE
Model.objects.all() does not convert types in primary keys

### ISSUE THREAD

**Issue by jacebrowning** [jacebrowning/datafiles#147]

**Comment by nielslerches** [comment]
Hi,

Is this issue still relevant?

If so, I'd like to take a crack at solving the issue, as I have a potential use case for this feature.

**Comment by jacebrowning** [comment]
@nielslerches yes, it's still an issue. You can confirm that by removing the `xfail` marker to see the test fail.

Let me know if you have any more questions!
